crates/modelardb_server/src/parser.rs

```diff
@@ -681,7 +695,7 @@ mod tests {
 
     #[test]
     fn test_tokenize_and_parse_create_model_table() {
-        let sql = "CREATE MODEL TABLE table_name(timestamp TIMESTAMP, field_one FIELD, field_two FIELD(10), tag TAG)";
+        let sql = "CREATE MODEL TABLE table_name(timestamp TIMESTAMP, field_one FIELD, field_two FIELD(10.5), tag TAG)";
```

Comment:
Can't this fail for locale da_DK if float parsing is locale-dependent?

Reply:
Based on the documentation it seems only `en_US` is supported as `f32`s can only be parsed if they follow this [grammar](https://doc.rust-lang.org/stable/std/primitive.f32.html#method.from_str).

Reply:
Based on the documentation it seems only `en_US` is supported as `f32`s can only be parsed if they follow this [grammar](https://doc.rust-lang.org/stable/std/primitive.f32.html#method.from_str).

Reply:
Good, this is not a problem then